common/config-manager.cpp

```diff
@@ -160,6 +162,7 @@ void ConfigManager::loadFromStream(SeekableReadStream &stream) {
 	_miscDomains.clear();
 	_transientDomain.clear();
 	_domainSaveOrder.clear();
+	//_sessionDomain.clear(); 
```

Comment:
Leftover?

Reply:
Yes, sorry. I didn't know if it was appropriate to clear it or not and in the end I forgot to remove the comment.